src/pages/tokens/[id].tsx

```diff
@@ -6,14 +6,24 @@ import { TokenData } from '@defichain/whale-api-client/dist/api/tokens'
 import { GetServerSidePropsContext, GetServerSidePropsResult, InferGetServerSidePropsType } from 'next'
 import { IoAlertCircleOutline, IoCheckmarkCircle } from 'react-icons/io5'
 import { Container } from '@components/commons/Container'
-import { AddressLink } from '@components/commons/AddressLink'
+import { AddressLink, AddressLinkExternal } from '@components/commons/AddressLink'
 import { TxIdLink } from '@components/commons/TxIdLink'
 import { isNumeric } from '../../utils/commons/StringValidator'
 
 interface TokenAssetPageProps {
   token: TokenData
 }
 
+enum TokensWithBackingAddress {
```

Comment:
With the switch statement this can be removed.

Reply:
well, it is also used here https://github.com/DeFiCh/scan/blob/a7535e69e17cb554ce3861ae9fc90efee36838e7/src/pages/tokens/%5Bid%5D.tsx#L112-L116